roles/openshift_node_group/tasks/create_config.yml

```diff
@@ -39,7 +39,7 @@
     yedit:
       content:
       src: "{{ mktempout.stdout }}/node-config.yaml"
-      edits: "{{ openshift_node_group_edits }}"
+      edits: "{{ openshift_node_group_edits | union(openshift_node_labels_edit) }}"
```

Comment:
Why do this?

Reply:
labels were not being set if a config map already existed.  They were only being set in the template file.

This adds an edit so that the labels are configured both edits, while not changing the format of the openshift_node_group variable.

I think it would be better to remove the "labels" from the openshift_node_group[0], but that might mess with existing data.

Reply:
How would these already exist?  I'm confused about the conditions that present this problem.

Does this affect a new cluster?  Upgrade?  Trying to re-run some portion of code against existing cluster?